package-lock.json

```diff
@@ -7188,6 +7188,14 @@
       "integrity": "sha512-iyam8fBuCUpWeKPGpaNMetEocMt364qkCsfL9JuhjXX6dRnguRVOfk2GZaDpPjcOKiiXCPINZC1GczQ7iTq3Zw==",
       "dev": true
     },
+    "ng2-codemirror": {
+      "version": "1.1.3",
+      "resolved": "https://registry.npmjs.org/ng2-codemirror/-/ng2-codemirror-1.1.3.tgz",
+      "integrity": "sha512-aqBTOTCo1poOMKxh1hil4ehCN7NgnaxGKG0Hu6Mwe9ya7yN3SQpPUCznw0kWBdN8ir1esbn13vh8y1bSewim4A==",
+      "requires": {
+        "codemirror": "5.48.0"
+      }
+    },
```

Comment:
Did you end up needing manual changes to this file? Or is it just the output from having run `npm install`?

Reply:
I ran npm install and it did two things:

1- Added the changes that you see here to the file.

2- It removed all of the ^ from the file.

Therefore, I just added #1 manually.